kotlin-semver/src/jsMain/generated/semver/functions/inc/IdentifierBase.kt

```diff
@@ -0,0 +1,15 @@
+// Generated by Karakum - do not modify it manually!
+
+package semver.functions.inc
```

Comment:
`inc` looks like redundant subpackage

Reply:
Well, it is the package-strategy `inc` namespace you two agreed on Discord is the best solution for the case.
I guess it is intended to have separate namespace from the other functions in case they als need some classes or typedefs. Do you think differently now how it should be solved?

Reply:
It's my fault :(
I thought, that redundand package won't be created